config_repo/config.sh.repo

```diff
@@ -159,10 +161,12 @@ POST_END_OF_NIGHT_DATA="false"
 # You must first create dark frames - see the README.md file for instructions.
 DARK_FRAME_SUBTRACTION="false"
 
-# If you have a fan in your Allsky camera enclosure that reports its speed,
-# set this to the full pathname of that file.
-# The file should include two numbers: the fan speed in RPM and the percent of maximum speed.
-FAN_DATA_FILE=""
+# If you have additional data or buttons you want displayed on the WebUI's System page,
+# specify one or more colon-separated files that contain that data.
+# You must provide scripts to update those files.
+# See the WEBUI_DATA_FILES configuration variable in the "Software settings" section of
+# https://github.com/thomasjacquin/allsky/wiki/allsky-Settings for details.
+WEBUI_DATA_FILES=""
```

Comment:
Looks like you've based this branch out of another one instead of `master` but that shouldn't be a problem. You may end up with a merge conflict but it should resolve easily